UnitTests/expected/Storyboards-osx-Message-Defaults.swift.out

```diff
@@ -46,22 +46,22 @@ struct StoryboardScene {
 
     case MessageDetailsScene = "MessageDetails"
     static func instantiateMessageDetails() -> NSViewController {
-      return StoryboardScene.Message_Osx.MessageDetailsScene.controller()
+      return StoryboardScene.Message_Osx.MessageDetailsScene.controller() as! NSViewController
```

Comment:
We should avoid force-casts (`as!`) and use `guard let … as? … else { fatalError("some explicit message") }` instead, like I did recently on `master`.

BTW, could you confirm me that swiftlint warns you about this when you run your Unit Tests and complain that you shouldn't use `as!`? I added the SwiftLint Build Script Phase recently to automatically check code style even in the templates, but not sure if it does work in your computer or only on mine*

* you must [install swiftlint on your computer](https://github.com/realm/SwiftLint/blob/master/README.md) for the Build Scripts to run and warn you about those, though